requirements.txt

```diff
@@ -2,6 +2,7 @@ Django==1.9
 celery==3.1.20
 djangorestframework==3.4
 django-filter==1.0.1
+pandas==0.24.2
```

Comment:
@prasoonmayank Can you use the builtin CSV library instead of pandas? It doesn't seem justified to require pandas for this operation.

Reply:
Handled